### ISSUE TITLE
NTR: Salty taste to skin, inappropriate (Skin and mucosa)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8036]
New term request Salty taste to skin, inappropriate (segal_210722190619): Not after heat or exercise; salt crystals on skin included

**Comment by pnrobinson** [comment]
@MickeySegal
I think this finding intends to be for cystic fibrosis, and as such it is very close to
[Elevated sweat chloride HP:0012236](https://hpo.jax.org/app/browse/term/HP:0012236)
I am not sure if it is something that is used very often and maybe it is better to make this a RELATED SYNONYM of that term? Or is there a separate diagnostic value?

**Comment by MickeySegal** [comment]
We have separate findings for clinical findings and lab findings.  Here the clinical finding doesn't tell you about chloride per se, but is due to elevated sodium chloride as ascertained by history or examination.  In practice, asking whether someone has salt crystals on their skin when others don't have crystals is useful in getting information before a lab test.  Not all such crystals are due to cystic fibrosis; some seem to be other NaCl dumping conditions with no CFTR variants and no clinical symptoms.  

More generally, dumping NaCl and thereby reducing the symptoms of ADHD may be the heterozygote advantage of CF, but other non-CFTR genes manage to get this without a disease phenotype.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000725
- label: Salty tasting skin
- parents: [Abnormality of skin physiology (HP:0011122)](https://hpo.jax.org/app/browse/term/HP:0011122)
- pmid: PMID:25640976
- def: Skin tastes marked salty without an obvious explanation such as heat or exercise. This finding can be observed in infants with cystic fibrosis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8036
- synonyms:
